evmcore/tx_validation_test.go

```diff
@@ -245,47 +132,156 @@ func TestValidateTx_GasPriceAndTip_RejectsTxWith(t *testing.T) {
 	}
 }
 
-func TestValidateTx_Gas_RejectsTxWith(t *testing.T) {
+func TestValidateTxStatic_Blobs_RejectsTxWith(t *testing.T) {
+	// blob txs are not supported in sonic, so they must have empty hash list and sidecar
+
+	t.Run("blob tx with non-empty blob hashes", func(t *testing.T) {
+		tx := types.NewTx(makeBlobTx([]common.Hash{{0x01}}, nil))
+		err := ValidateTxStatic(tx)
+		require.ErrorIs(t, err, ErrTxTypeNotSupported)
+	})
+
+	t.Run("blob tx with non-empty sidecar", func(t *testing.T) {
+		tx := types.NewTx(makeBlobTx(nil,
+			&types.BlobTxSidecar{Commitments: []kzg4844.Commitment{{0x01}}}))
+		err := ValidateTxStatic(tx)
+		require.ErrorIs(t, err, ErrTxTypeNotSupported)
+	})
+}
+
+func TestValidateTxStatic_AuthorizationList_RejectsTxWith(t *testing.T) {
+	t.Run("setCode tx with empty authorization list", func(t *testing.T) {
+		tx := types.NewTx(&types.SetCodeTx{})
+		err := ValidateTxStatic(tx)
+		require.ErrorIs(t, err, ErrEmptyAuthorizations)
+	})
+}
+
+func TestValidateTxStatic_ReturnsNilToAcceptableTx(t *testing.T) {
```

Comment:
nit: less technical, more semantic
```suggestion
func TestValidateTxStatic_AcceptsValidTransactions(t *testing.T) {
```